js-module-base/pom.xml

```diff
@@ -80,12 +82,12 @@
                         <version>1.8</version>
                         <executions>
                             <execution>
-                                <phase>initialize</phase>
+                                <phase>package</phase>
                                 <configuration>
                                     <target>
                                         <!-- Use the pluginId as the asset namespace. -->
                                         <copy todir="${project.build.directory}/assets-wrapper/assets/${project.artifactId}/jsmodules">
-                                            <fileset dir="${project.basedir}/src/main/webapp/jsmodules"/>
+                                            <fileset dir="${basedir}/src/main/webapp/jsmodules"/>
```

Comment:
Why? IIRC you are supposed to use `project.basedir`, not `basedir`.

Reply:
Same as above ... `project.basedir` didn't work (no files were copied to the expected location) while `basedir` did work.

Reply:
So this is the error ....

```
${project.basedir} expression not supported during profile activation, use ${basedir} instead
```